target/linux/ramips/image/mt7620.mk

```diff
@@ -79,6 +79,18 @@ define Device/wn3000rpv3
 endef
 TARGET_DEVICES += wn3000rpv3
 
+define Device/ex3700
```

Comment:
It is, but currently the order is: ex2700, wn3000rpv3, ex3700. And that seems incorrect.